v63004/inref/gtm8202.m

```diff
@@ -31,5 +31,8 @@
 	 set ^c(4)="12"
 	 TCOMMIT
 
+	 ;ensures all updates to DB are moved from memory to disk
+	 ;mupip journal -extract only checks the disk
```

Comment:
mupip journal -extract --> mupip journal -extract (done in caller script right after this M program exits)"